install-panel.sh

```diff
@@ -590,16 +590,45 @@ function firewall_firewalld {
   echo "* Firewall-cmd installed"
   print_brake 70
 }
+function centos_based_letsencrypt {
+    if [ "$OS_VER_MAJOR" == "7" ]; then
+      # Enable EPEL falowing official guide https://fedoraproject.org/wiki/EPEL
+      yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Comment:
I am unsure if a conflict occurs here since the function `centos7_dep` also installs some form of epel-release package.